jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPartFormData.java

```diff
@@ -536,8 +546,21 @@ public void onPart(String name, String fileName, HttpFields headers)
                     if (headers.contains("content-transfer-encoding"))
                     {
                         String value = headers.get("content-transfer-encoding");
+                        if ("base64".equals(value))
+                            complianceViolationListener.onComplianceViolation(
+                                new ComplianceViolation.Event(MultiPartCompliance.RFC7578,
+                                    MultiPartCompliance.Violation.BASE64_TRANSFER_ENCODING,
+                                    value));
+                        if ("quoted-printable".equals(value))
```

Comment:
else-if instead?

Reply:
Done, but only for this if, the last if is separate (and a different violation enum)